src/applications/coronavirus-vaccination-expansion/config/va-location/DynamicRadioWidget.jsx

```diff
@@ -81,7 +81,7 @@ export function DynamicRadioWidget(props) {
           } ${location.attributes.state}`}</p>
         </>
       ),
-      value: location.attributes.name,
+      value: `${location.attributes.name}|${location.id}`,
```

Comment:
Cool solution :+1:

Reply:
Thanks @bkjohnson!  I was thinking about this later and wanted to see if an object could be used as value like `{name: loc.attr.name, id: loc.id}.  Would remove the need for the string.split.  Just have not had time to mess with it any further yet and not sure if value could take an object.